ci/jenkins/README.md

```diff
@@ -157,6 +157,15 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase networkpolicy --registry ${DOCKER_REGISTRY} --testbed-type "kind" --kind-cluster-name "${{JOB_NAME}}-${{BUILD_NUMBER}}"
 ```
 
+* "/stop-all": Trigger '/stop-all' to kill stale running or waiting jobs related to a PR.
```

Comment:
I think the current method only covers capv-related jobs. Do you plan to support other jobs running on the static testbed by another PR? If that is the case, please also update the README doc to avoid any misunderstandings.  @KMAnju-2021

Reply:
yes, currently i am thinking to enabling this feature only for capv-related jobs if it will work fine then I will enable it for other jobs that are running on the static testbed as well. not sure but do you have any comments on it?
I will update README, thanks!